gateway/main/settings.py

```diff
@@ -306,6 +306,8 @@
 # resources limitations
 LIMITS_JOBS_PER_USER = int(os.environ.get("LIMITS_JOBS_PER_USER", "2"))
 LIMITS_MAX_CLUSTERS = int(os.environ.get("LIMITS_MAX_CLUSTERS", "6"))
+LIMITS_CPU_PER_TASK = int(os.environ.get("LIMITS_CPU_PER_TASK", "4"))
+LIMITS_MEMORY_PER_TASK = int(os.environ.get("LIMITS_MEMORY_PER_TASK", "8"))
```

Comment:
How do these variable work?

Reply:
cpus / ram per task are set in the helm config. I added both as envvars on the gateway pod, and this code here just reads those values in. that way we can tell the user what the limits are if they request too many

Reply:
just a friendly ping here @akihikokuroda if you can take a look 🙏